plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPreferencesDialog.java

```diff
@@ -66,7 +66,7 @@ protected Control createDialogArea(final Composite parent) {
 
     Composite container = new Composite(dialogArea, SWT.NONE);
     content = new StandardDeployPreferencesPanel(container, project, loginService,
-        getLayoutChangedHandler(), false /* allowEmptyProjectId */);
+        getLayoutChangedHandler(), true /* requireProjectId */, true /* requireAccount */);
```

Comment:
realizing now that this could really be one flag: requireValues. I don't think the preference dialog requires anything to be set. What do you think?